import_flow_events.py

```diff
@@ -32,6 +32,13 @@
 EVENTS_PREFIX = "fxa-flow/data/"
 EVENTS_FILE_URL = "s3://" + EVENTS_BUCKET + "/" + EVENTS_PREFIX + "flow-{day}.csv"
 
+# The CSV format has changed over time. This config controls the
+# column names, definitions and deploy dates for those changes.
+ADDITIONAL_COLUMNS = (
+    {"name":"locale", "definition":"VARCHAR(40)", "deploy_date":"2017-02-21"},
+    {"name":"uid", "definition":"VARCHAR(64)", "deploy_date":"2017-02-21"},
```

Comment:
Fwiw, it turns out this deploy date is not correct. I assumed it would be from the merge, but the CSV for `2017-02-22` still contains 15 commas per line. I'll keep an eye on the CSVs then come back to update this as and when.